src/platform/EFR32/BLEManagerImpl.cpp

```diff
@@ -502,23 +508,31 @@ bool BLEManagerImpl::SendIndication(BLE_CONNECTION_OBJECT conId, const ChipBleUU
     sl_status_t ret;
     uint16_t cId        = (UUIDsMatch(&ChipUUID_CHIPoBLEChar_RX, charId) ? gattdb_CHIPoBLEChar_Rx : gattdb_CHIPoBLEChar_Tx);
     uint8_t timerHandle = GetTimerHandle(conId, true);
+    ChipDeviceEvent event;
 
     VerifyOrExit(((conState != NULL) && (conState->subscribed != 0)), err = CHIP_ERROR_INVALID_ARGUMENT);
     VerifyOrExit(timerHandle != kMaxConnections, err = CHIP_ERROR_NO_MEMORY);
 
-    // start timer for light indication confirmation. Long delay for spake2 indication
-    sl_bt_system_set_soft_timer(TIMER_S_2_TIMERTICK(6), timerHandle, true);
-
-    ret = sl_bt_gatt_server_send_indication(conId, cId, (data->DataLength()), data->Start());
+    // start timer for light notification confirmation. Long delay for spake2 indication
+    sl_bt_system_set_soft_timer(TIMER_S_2_TIMERTICK(10), timerHandle, true);
```

Comment:
Based on our converstions , is the longer delay from 6 to 10 still needed?

Reply:
No, it isn't. Changed it back to 6 ms.